### PR TITLE
chore(travel-dungeon): finish the 1800 times

### PR DESCRIPTION
**Pull request by Karguel1995**
It was a long work, but it's worth it :)

I created folders for rooms, because there are too many files. 
I checked all rooms, they are connected, and without blank/error pages.

### CODE REVIEW

File: english/time-travel/0/10/10-3B.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 You survived and killed werewolf 🐺! Now you can play another dungeons or rooms! 
 
-- [Start from beggining](../begin-journey.md)
+- [Start from beggining](../../begin-journey.md)
```
Comment (AvidCoder101) [inline english/time-travel/0/10/10-3B.md:5]: Can you please fix this typo : beggining to beginning?

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]

File: english/time-travel/0/10/10-3B.md (hunk 1 above)
Comment (Karguel1995) [reply english/time-travel/0/10/10-3B.md:5]: I am on it already! Thanks!

Comment (Karguel1995) [reply english/time-travel/0/10/10-3B.md]: Done!
If you see more words/sentences to change, I will really appreciate if you share with me! :)

**Review by AvidCoder101 — APPROVED** [review]
LGTM, thanks for changing the typos.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**